config/locales/client.en.yml

```diff
@@ -4136,6 +4130,14 @@ en:
                 one: "%{count} draft"
                 other: "%{count} drafts"
 
+    bootstrap_mode_banner:
+      enabled:
+        one: "To make launching your new site easier, you are in bootstrap mode. All new users will be granted trust level 1 and have daily email summary emails enabled. This will be automatically turned off when %{count} user has joined."
+        other: "To make launching your new site easier, you are in bootstrap mode. All new users will be granted trust level 1 and have daily email summary emails enabled. This will be automatically turned off when %{count} users have joined."
+      disabled: "Bootstrap mode will be disabled within 24 hours."
```

Comment:
Minor, but AFAICT the three strings above haven't changed, so maybe we should keep the existing translation keys, to avoid having extra work for translators or for overrides to no longer apply.

Reply:
Agreed, made the required changes.